src/Microsoft.PowerShell.Commands.Utility/Microsoft.PowerShell.Commands.Utility.csproj

```diff
@@ -78,6 +78,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="2.6.1" />
+    <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic" Version="2.6.1" />
```

Comment:
I remember we explicitly decided not to support VisualBasic in pwsh implementation because it's not used and increases the size of the package.

Reply:
In previous cleanup PR we got PowerShell Committee conclusion about Version type - nothing has been said that VisualBasic  is not necessary 😕 Also we have request about adding F#.. And JS (it seems impossible).

Reply:
Could we have a design that added languages as a package?

Another potential issue with adding languages is greater exposure to security issues.

Reply:
I could not make a common code for C# and VB - each language requires its own code path. So I have no idea how we can add languages as a package.